Explorer/Assets/DCL/Character/CharacterMotion/Velocity/ApplyThirdPersonRotation.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DCL.CharacterMotion.Components;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+namespace DCL.Character.CharacterMotion.Velocity
+{
+    public static class ApplyThirdPersonRotation
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Execute(ref CharacterRigidTransform rigidTransform, in MovementInputComponent input)
+        {
+            if (!(Mathf.Abs(input.Axes.x) > 0) && !(Mathf.Abs(input.Axes.y) > 0))
```

Comment:
Is it not simpler avoidng the two NOT's?

if (Mathf.Abs(input.Axes.x) < Mathf.Epsilon && Mathf.Abs(input.Axes.y) < Mathf.Epsilon)
    return;